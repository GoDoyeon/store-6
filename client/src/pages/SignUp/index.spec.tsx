import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import SignUpPage from './index';

describe('<SignUpPage />', () => {
  it('should render same with snapshot', () => {
    const { container } = render(<SignUpPage />);
    expect(container).toMatchSnapshot();
  });
});
