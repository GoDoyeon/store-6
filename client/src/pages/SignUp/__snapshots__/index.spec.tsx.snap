// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUpPage /> should render same with snapshot 1`] = `
.c19 {
  position: relative;
  height: 55px;
  width: 183px;
  font-size: 25px;
  background: center / cover no-repeat url();
  cursor: pointer;
}

.c14 {
  position: relative;
  height: 16px;
}

.c16 {
  height: 100%;
}

.c15 {
  position: absolute;
  bottom: 5px;
  left: 1px;
}

.c20 {
  margin-top: 28px;
  font-style: normal;
  font-weight: normal;
  font-size: 12px;
  line-height: 16px;
  color: #999999;
}

.c9 {
  height: 48px;
  content: '';
}

.c21 {
  height: 32px;
  content: '';
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100vh;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
}

.c22 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100vh;
  width: 33%;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 2;
  -ms-flex: 2;
  flex: 2;
  min-width: 400px;
  max-width: 400px;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  overflow-y: scroll;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-top: 64px;
  margin-bottom: 64px;
  font-size: 48px;
}

.c18 {
  margin-top: 42px;
}

.c12 {
  width: 100%;
}

.c13 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 10px;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  margin-top: 32px;
  padding-left: 14px;
  cursor: pointer;
}

.c8 {
  box-sizing: border-box;
  width: 100%;
  height: 34px;
  padding: 0 8px 0 8px;
  line-height: 25px;
  font-size: 20px;
  background: bottom left / contain no-repeat url();
}

.c8::-webkit-input-placeholder {
  color: #999;
}

.c8::-moz-placeholder {
  color: #999;
}

.c8:-ms-input-placeholder {
  color: #999;
}

.c8::placeholder {
  color: #999;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  gap: 32px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
}

.c6 {
  padding-left: 8px;
  font-size: 20px;
  line-height: 25px;
}

.c7 {
  font-size: 12px;
  line-height: 15px;
  color: #ff9e2c;
}

.c17 {
  display: block;
  margin-top: 8px;
  border: 1px solid #eeeeee;
  box-sizing: border-box;
  border-radius: 10px;
  overflow-y: scroll;
  width: 100%;
  padding: 16px;
  box-sizing: border-box;
  height: 80px;
  font-family: 'noto sans',sans-serif !important;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  margin-left: -16px;
  width: 100%;
}

.c11 {
  text-align: center;
  margin: 0 16px;
  width: 110px;
  box-sizing: border-box;
  height: 34px;
  padding: 0 8px 0 8px;
  line-height: 25px;
  font-size: 20px;
  background: bottom left / contain no-repeat url();
}

.c11::-webkit-input-placeholder {
  color: #999;
}

.c11::-moz-placeholder {
  color: #999;
}

.c11:-ms-input-placeholder {
  color: #999;
}

.c11::placeholder {
  color: #999;
}

<div>
  <div
    class="c0"
  >
    <aside
      class="c1"
    />
    <form
      class="c2"
    >
      <header
        class="c3"
      >
        <img
          alt="robot"
        />
        <h1
          class="text-baemin100"
        >
          배민
        </h1>
        <h1>
          문방구
        </h1>
      </header>
      <div
        class="c4"
      >
        <div
          class="c5"
        >
          <label
            class="c6"
          >
            아이디
          </label>
          <span
            class="c7"
          >
            중복된 아이디입니다.
          </span>
        </div>
        <input
          autocomplete="off"
          class="c8"
          id="id"
          name="id"
          type="text"
        />
      </div>
      <div
        aria-hidden="true"
        class="c9"
        height="48px"
      />
      <div
        class="c4"
      >
        <div
          class="c5"
        >
          <label
            class="c6"
          >
            비밀번호
          </label>
          <span
            class="c7"
          >
            비밀번호 다시 확인할것!
          </span>
        </div>
        <input
          class="c8"
          id="password"
          name="password"
          type="password"
        />
      </div>
      <div
        aria-hidden="true"
        class="c9"
        height="48px"
      />
      <div
        class="c4"
      >
        <div
          class="c5"
        >
          <label
            class="c6"
          >
            비밀번호 확인
          </label>
          <span
            class="c7"
          >
            비밀번호가 일치하지 않습니다.
          </span>
        </div>
        <input
          class="c8"
          id="password-retype"
          name="password-retype"
          type="password"
        />
      </div>
      <div
        aria-hidden="true"
        class="c9"
        height="48px"
      />
      <div
        class="c4"
      >
        <div
          class="c5"
        >
          <label
            class="c6"
          >
            이메일
          </label>
          <span
            class="c7"
          >
            이메일 형식이 맞지 않습니다.
          </span>
        </div>
        <input
          class="c8"
          id="email"
          name="email"
          type="email"
        />
      </div>
      <div
        aria-hidden="true"
        class="c9"
        height="48px"
      />
      <div
        class="c4"
      >
        <div
          class="c5"
        >
          <label
            class="c6"
          >
            전화번호
          </label>
          <span
            class="c7"
          >
            전화번호 형식이 맞지 않습니다.
          </span>
        </div>
        <div
          class="c10"
        >
          <input
            class="c11"
            id="010"
            name="010"
            type="text"
          />
          <img
            alt="hyphen"
          />
          <input
            class="c11"
            id="1234"
            name="1234"
            type="text"
          />
          <img
            alt="hyphen"
          />
          <input
            class="c11"
            id="5678"
            name="5678"
            type="text"
          />
        </div>
      </div>
      <div
        aria-hidden="true"
        class="c9"
        height="48px"
      />
      <div
        class="c5"
      >
        <label
          class="c6"
        >
          약관 동의
        </label>
      </div>
      <section
        class="c12"
      >
        <div
          class="c13"
        >
          <div
            class="c14"
          >
            <img
              class="c15"
            />
            <img
              class="c16"
            />
          </div>
          <span
            style="font-size: 20px; line-height: 25px;"
          >
            아래 약관에 모두 동의합니다
          </span>
        </div>
      </section>
      <section
        class="c12"
      >
        <div
          class="c13"
        >
          <div
            class="c14"
          >
            <img
              class="c15"
            />
            <img
              class="c16"
            />
          </div>
          <span>
            개인정보 이용약관 (필수)
          </span>
        </div>
      </section>
      <div
        class="c17"
      >
        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nobis doloremque quam quasi cupiditate suscipit optio, aliquid vero omnis ducimus? Nulla, laudantium maxime incidunt laborum iure vero consequuntur tenetur voluptatibus voluptate.
      </div>
      <section
        class="c12"
      >
        <div
          class="c13"
        >
          <div
            class="c14"
          >
            <img
              class="c15"
            />
            <img
              class="c16"
            />
          </div>
          <span>
            배민문방구 이용약관 (필수)
          </span>
        </div>
      </section>
      <div
        class="c17"
      >
        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nobis doloremque quam quasi cupiditate suscipit optio, aliquid vero omnis ducimus? Nulla, laudantium maxime incidunt laborum iure vero consequuntur tenetur voluptatibus voluptate.
      </div>
      <div
        class="c18"
      >
        <button
          class="c19"
        >
          회원가입
        </button>
      </div>
      <footer
        class="c20"
      >
        COPYRIGHT © 2021 우아한형제들 ALL RIGHTS RESERVED.
      </footer>
      <div
        aria-hidden="true"
        class="c21"
        height="32px"
      />
    </form>
    <aside
      class="c22"
    />
  </div>
</div>
`;
